<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovoRegistro</class>
 <widget class="QWidget" name="NovoRegistro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/png/globe.png</normaloff>../assets/png/globe.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0D192B;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: #0D192B;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="lbl_Titulo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Adicionando um novo registro</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lbl_Nome">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(219, 219, 219);</string>
              </property>
              <property name="text">
               <string>Nome:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="input_Nome">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	color: rgb(219, 219, 219);
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
	border: 1px solid rgb(219, 219, 219);;
	border-radius: 6px;
}

QLineEdit:hover{
	color: rgb(219, 219, 219);
	min-height: 40px;
	max-height: 40px;
	min-width:500px;
	max-width:500px;
	border: 1px solid rgb(219, 219, 219);;
	border-radius: 6px;
	background-color: #081221
}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="lbl_Tipo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(219, 219, 219);</string>
              </property>
              <property name="text">
               <string>Tipo:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbox_Tipo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
    background-color: #0D192B;
    color: #DBDBDB;
    border: 1px solid white;
    border-radius: 6px;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
}

QComboBox QAbstractItemView {
    selection-color: #0D192B;
    border: 1px solid #2E3B55;
	border-radius:6px;
	background-color: #DBDBDB;
    color: rgb(13, 25, 43);
    min-width: 100px;  /* &lt;-- isso evita o corte */
    padding: 5px;
    selection-background-color:#C9C9C9;
	color: rgb(13, 25, 43);
}

QComboBox:hover{
    color: #DBDBDB;
    border: 1px solid white;
    border-radius: 6px;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
	background-color: #081221
}</string>
              </property>
              <item>
               <property name="text">
                <string>Entrada</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saída</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lbl_Categoria">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(219, 219, 219);</string>
              </property>
              <property name="text">
               <string>Categoria:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbox_Categoria">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
    background-color: #0D192B;
    color: #DBDBDB;
    border: 1px solid white;
    border-radius: 6px;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
}

QComboBox QAbstractItemView {
    selection-color: #0D192B;
    border: 1px solid #2E3B55;
	border-radius:6px;
	background-color: #DBDBDB;
    color: rgb(13, 25, 43);
    min-width: 100px;  /* &lt;-- isso evita o corte */
    padding: 5px;
    selection-background-color:#C9C9C9;
	color: rgb(13, 25, 43);
}

QComboBox:hover{
    color: #DBDBDB;
    border: 1px solid white;
    border-radius: 6px;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
	background-color: #081221
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="lbl_Data">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(219, 219, 219);</string>
              </property>
              <property name="text">
               <string>Data:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="input_Data">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QDateEdit {
    background-color: #0D192B;
	border: 1px solid #DBDBDB;
	border-radius: 6px;
	color: #DBDBDB;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
}

QDateEdit:hover{
	border: 1px solid #DBDBDB;
	border-radius: 6px;
	color: #DBDBDB;
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
	background-color: #081221
}
</string>
              </property>
              <property name="displayFormat">
               <string>dd/MM/yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="lbl_Valor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(219, 219, 219);</string>
              </property>
              <property name="text">
               <string>Valor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="input_Valor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>502</width>
                <height>52</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	color: rgb(219, 219, 219);
	min-height: 50px;
	max-height: 50px;
	min-width:500px;
	max-width:500px;
	border: 1px solid rgb(219, 219, 219);;
	border-radius: 6px;
}

QLineEdit:hover{
	color: rgb(219, 219, 219);
	min-height: 40px;
	max-height: 40px;
	min-width:500px;
	max-width:500px;
	border: 1px solid rgb(219, 219, 219);;
	border-radius: 6px;
	background-color: #081221
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_10">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Confirmar">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: #147117;
	color: rgb(255, 255, 255);
	color: rgb(255, 255, 255);
	padding: 10px 20px;
	border-radius: 12px;
	min-width: 150px;
	min-height: 30px;
	font-weight: bold;
}

QPushButton:hover {
	background-color: #116514;
	padding-top: 12px;
}
                          
QPushButton:pressed {
	 background-color: #0D570F;
	padding-top: 14px;
}</string>
              </property>
              <property name="text">
               <string>Confirmar</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
